### DISCUSSION TITLE
Automatically set an article to be read after scolling it out of view

### DISCUSSION THREAD

**Discussion by Kraftwurm** [SimonHalvdansson/Harmonic-HN#182] [Ideas]
My pattern of using Harmonic-HN ist always the following:

- Start at the top
- Check every article and decide to either read it or mark it as read with the corresponding context menu
- repeat until I have enough or reach the bottom = no more unread articles

Because only ~10% of all articles really interest me on average I'm constantly using the context menu to "Mark As Read" 

Would it be possible to have a "auto mark as read" feature in the seetings?

Thanks for the great tool! I use it every day.

**Comment by SimonHalvdansson** [comment]
Not a bad idea, I think I like it. https://github.com/SimonHalvdansson/Harmonic-HN/issues/314
